app/src/main/java/com/example/mytrainingpal/screens/Calendar.kt

```diff
@@ -8,24 +8,28 @@ import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
```

Comment:
Here my AndroidStudio screams that it can't find it. It works nonetheless, but maybe you know how to make it better?